src/menu.ts

```diff
@@ -44,7 +45,10 @@ export function setupMenu( mainWindow: BrowserWindow | null ) {
 			label: app.name, // macOS ignores this name and uses the name from the .plist
 			role: 'appMenu',
 			submenu: [
-				{ role: 'about' },
+				{
+					label: 'About Studio',
```

Comment:
Let's translate this label.

```suggestion
					label: __( 'About Studio' ),
```

Reply:
Done in https://github.com/Automattic/studio/pull/104/commits/e03743c3df30f999262458e2301e29bc9a84c0ce